src/clj/game/core/costs.clj

```diff
@@ -136,10 +157,15 @@
 ;; X Credits
 (defmethod cost-name :x-credits [_] :x-credits)
 (defmethod value :x-credits [_] 0)
+;We put stealth credits in the third slot rather than the empty second slot for consistency with credits
+(defmethod stealth-value :x-credits [[_ __ stealth-amount]] (or stealth-amount 0))
+;Do not merge x-credit costs, if an ability somehow gets 2, each should be handled seperately
+(defmethod merge-cost :x-credits [cost1 cost2] [cost1 cost2])
```

Comment:
I don't think this works. `merge-cost` should be returning a vector with 2 elements: cost-type, and cost-value. This will return a vector of two cost vectors.